### ISSUE TITLE
"Use your SIP account" problems

### ISSUE THREAD

**Issue by mixaz** [BelledonneCommunications/linhome-android#29]
There are few issues in input screens when registering external SIP account (sip.linphone.org). It is possible to add that account but the way is not clear.

1) After entering username, sip domain (sip.linhome.org) and password and pressing "Login" button, nothing happens

2) In "Advanced settings" proxy field  is empty and market "Mandatory". Entering just sip.linphone.org doesn't work, only full form form with username, ie sip:gmixaz@sip.linphone.org, that is not quite clear.

**Comment by JonathanBartet** [comment]
Hi Mixaz,

Dou you still have this issue? Just tried and i didn't reproduce.

**Comment by mixaz** [comment]
Yes, the issue it remains - account at linhome.org is registered OK but SIP connection is reported as "Connection failed", on "My Account" screen.

**Comment by mixaz** [comment]
I confirm - the issue remains. I can't login without entering "more settings" section and specifying "proxy" field. Pressing "login" button doesn't show any error, so user has to guess what's going wrong, and correct that in "more settings" section.

**Comment by mixaz** [comment]
test case:

1. Menu / Account Assistant / Use your SIP account
2. username: gmixaz, sip domain: sip.linphone.org, password <my password>
3. Press "login" button. Nothing happens

App version: 5.0.0 version code 26 (debug)

I built the app from source code in https://gitlab.linphone.org/BC/public/linhome-android. I didn't install it from GP

**Comment by mixaz** [comment]
Tested in GP version of linhome app - works OK, in the test case above I do not need to enter "Advanced Settings" screen.

Low priority issue

**Comment by gcartlidge** [comment]
Thanks @mixaz , this has been driving me crazy while doing a customised apk for my company, adding the full account name under proxy allows me to register!
